Utils/src/TriggerProducer.cc

```diff
@@ -89,9 +94,15 @@ TriggerProducer::TriggerProducer(const edm::ParameterSet& iConfig)
   trigResultsTok_ = consumes<edm::TriggerResults>(trigResultsTag_);
   trigPrescalesTok_ = consumes<pat::PackedTriggerPrescales>(trigPrescalesTag_);
 
+  edm::InputTag theTriggerLabel("TriggerResults", "", "HLT");
+  edm::InputTag theTrigObjLabel("selectedPatTrigger");
+  trigResultsToken = consumes<edm::TriggerResults>(theTriggerLabel);
+  trigObjCollToken = consumes<pat::TriggerObjectStandAloneCollection>(theTrigObjLabel);
+
   produces<std::vector<std::string> >("TriggerNames");
   produces<std::vector<int> >("TriggerPass");
   produces<std::vector<int> >("TriggerPrescales");
+  produces<std::vector<TLorentzVector> >("HLTElectronObjects");
```

Comment:
add a configurable bool so it doesn't produce this all the time, only when requested

Reply:
did you mean something like this?
bool saveHLTObj = true or false;
if(saveHLTObj) produces<std::vector<TLorentzVector> >("HLTElectronObjects");

this bool will be used in the next part of the code as well.

Reply:
yes, and it should be set by the python config for the module.